File: 1979-find-greatest-common-divisor-of-array/1979-find-greatest-common-divisor-of-array.py
class Solution:
    def findGCD(self, nums: List[int]) -> int:
        
        maxNumber = max(nums)
        minNumber = min(nums)
        
        for j in range(minNumber,0,-1):
            if maxNumber % j == 0 and minNumber % j == 0:
                return j

File: 1979-find-greatest-common-divisor-of-array/1979-find-greatest-common-divisor-of-array.java
class Solution {
    public int findGCD(int[] nums) {
        Arrays.sort(nums);
        for (int i = nums[0]; i > 0;i--){
            if (nums[nums.length-1] % i == 0 && nums[0] % i == 0){
                return i;
            }
        }
        return 1;  
    }
}